src/main/16-cloudwatch-alarms.tf

```
@@ -0,0 +1,87 @@
+
+
+resource "aws_cloudwatch_metric_alarm" "apigw_server_errors" {
+  alarm_name          = "${var.app_name}-cwalarm-apigw-server-errors-${var.env}"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = 10
+  metric_name         = "5XXError"
+  namespace           = "AWS/ApiGateway"
+  period              = 60
+  statistic           = "SampleCount"
+  threshold           = 1
+  alarm_actions       = [aws_sns_topic.cw_alarms.arn]
+}
+
+resource "aws_cloudwatch_log_metric_filter" "error_logs" {
+
+  name           = "${var.app_name}-error-logs-filter-${var.env}"
+  pattern        = "ERROR"
+  log_group_name = "/aws/eks/interop-probing-eks-dev/application"
+
+  metric_transformation {
+    name      = "ErrorCount"
+    namespace = "ApplicationLogs"
+    value     = 1
+  }
```

Just a suggestion to organize metrics
```suggestion
  metric_transformation {
    name      = "ErrorCount"
    namespace = "EKSApplicationLogsFilters"
    value     = "1"

    dimensions = {
      PodApp = "$.pod_app"
    }
  }
```

---

dimensions not supported